### ISSUE TITLE
Set artwork rotation parameter in GUI

### ISSUE THREAD

**Issue by dancleaver** [MickHarrigan/uhdrtz-rust#46]
This would allow us to input something like the number of slices, avoiding the need to exit, hardcode a new value, and re-run.

**Comment by clostoski** [comment]
This is the same issue as the adjustable max interval issue. the max interval should be 360/ num slices. Having a drop down menu or an input box for number of slices. My thought is that it should go in the start up menu to limit the amount of math that is happening during run time.

**Comment by MickHarrigan** [comment]
> My thought is that it should go in the start up menu to limit the amount of math that is happening during run time.

Agreed. The math to convert between "slices" to the constant rotation amount is as follows...
```
C = (360 * PI) / (180 * slices)
```

This should be calculated within the cleanup_menu system (or something else running on exit of the setup state) and saved as a resource that can be read from by any of the systems that use this value.

This includes the use in `src/plugin.rs` where there is a line that edits the FixedTime resource to be related to this value.

**Comment by dancleaver** [comment]
> ```
> C = (360 * PI) / (180 * slices)
> ```
Let's just call it `2 * PI / slices`

**Comment by MickHarrigan** [comment]
Whoops I totally missed that one

**Comment by MickHarrigan** [comment]
Update:
Made a way to enter an amount of slices for the system to use, if it fails to detect a valid number it will default back to 24 fps.

In testing with higher framerates the overall rotational speed has lowered. I think that adding another factor that is just the overall rotational speed could be in order.
@dancleaver I would like to know your view on this and what the math would potentially look like, or if you would like to take a stab at it with me.
